packages/browser/src/plugins/page-enrichment/__tests__/index.test.ts

```diff
@@ -212,53 +212,6 @@ describe('Page Enrichment', () => {
   })
 })
 
-describe('pageDefaults', () => {
-  const el = document.createElement('link')
-  el.setAttribute('rel', 'canonical')
-
-  beforeEach(() => {
-    el.setAttribute('href', '')
-    document.clear()
-  })
-
-  afterEach(() => {
-    jest.restoreAllMocks()
-  })
-
-  it('handles no canonical links', () => {
-    const defs = pageDefaults()
-    expect(defs.url).not.toBeNull()
-  })
-
-  it('handles canonical links', () => {
```

Comment:
FYI: This PR does not remove any tests, it just moves a few around. For example, these are moved to: packages/browser/src/core/page/__tests__/index.test.ts